### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kalman**
Add a bit more info on squash merges and perf tests.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -49,5 +49,16 @@ For JS code, We have a python script to run all js tests.
...
+
+By default, neither `go test` nor Jenkins run the perf tests, because they take a while.
+
+To run the tests yourself, use the `-perf` and `-v` flag to `go test`, e.g.:
+
+* `go test -v ./samples/go/csv/... -perf mem`
+
+See https://godoc.org/github.com/attic-labs/noms/go/perf/suite for full documentation and flags.
+
+To ask Jenkins to run the perf tests for you, reply (no quotes) "Perf this" to your PR. Yes, really.
```
Comment (mikegray) [inline CONTRIBUTING.md]: Can you add a note about ability to use "test this" for getting Jenkins to rebuild the PR too?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,12 +26,12 @@ Due to legal reasons, all contributors must sign a contributor license agreement
 We follow a code review protocol dervied from the one that the [Chromium team](https://www.chromium.org/) uses:
 
 1. Create a fork of the repo you want to modify (e.g., fork `https://github.com/attic-labs/noms` to `https://github.com/<you>/noms`.
-2. Push your changes to a branch at your fork
+2. Push your changes to a branch at your fork.
 3. Create a PR using that branch against, e.g., `noms/master`.
 4. When you're ready for review, use Github's _assign_ UI to assign someone to review. Typically nobody will review until you assign someone (because we assume you're still getting it ready for review).
 5. Reviewer will make comments, then say either 'LGTM' (looks good to me) or 'BTY' (back to you).
 6. If the reviewer said LGTM, it means it is ready to merge. If you have commit writes to the respository, go ahead and land the PR. Otherwise the reviewer will land it.
```
Comment (mikegray) [inline CONTRIBUTING.md]: "commit writes" should be "commit rights" or "commit privileges"

File: CONTRIBUTING.md (hunk 1 above)
Comment (kalman) [inline CONTRIBUTING.md]: Done

File: CONTRIBUTING.md (hunk 2 above)
Comment (kalman) [inline CONTRIBUTING.md]: Done

**Comment by kalman** [comment]
@aboodman PTAL

**Comment by kalman** [comment]
(thanks @mikegray )

**Comment by aboodman** [comment]
lgtm

**Comment by aboodman** [comment]
lgtm

**Pull request merged**